ingestion/src/metadata/data_quality/runner/base_test_suite_source.py

```diff
@@ -122,7 +113,6 @@ def create_data_quality_interface(self) -> TestSuiteInterface:
             source_config_type=self.service_conn_config.type.value,
```

Comment:
Can you move these to their own methods? `self.get_sampler` and ``self.get_validator_class`

Reply:
On the collate side we can then just modify these two

Reply:
The importer injection [is achieved here](https://github.com/open-metadata/openmetadata-collate/pull/923/files#diff-cb6c2264a7b5993fb58a79a09e87c3a662c3decde41d13a32dc4581b66fdfa62R34).

Reply:
and [here](https://github.com/open-metadata/openmetadata-collate/pull/923/files#diff-0fa83b3b608553c1bd66e2707132b47d4aed1ef4ec81f9ad61c0789db36b6416R18-R21)